src/components/Configure/content/fields/RequiredFields.tsx

```diff
@@ -12,7 +12,7 @@ export function RequiredFields() {
 
   return (
     <>
-      <FieldHeader string={`${appName} reads the following ${selectedObjectName} fields`} />
+      <FieldHeader string={`${appName} reads the following fields`} />
```

Comment:
removed it, since we weren't using display name of the object, so it looked weird:
<img width="977" alt="Screenshot 2025-04-18 at 4 23 58 PM" src="https://github.com/user-attachments/assets/ba364c88-76d4-419e-9f6a-4edf4a0ee31d" />